Description: Use VFPv3-D16 FPU for ARM builds
 Jani writes: The D16 part was Debian/Ubuntu specific, IIRC we define hardfloat
 in that particular variant (16 double registers) or we had a different naming
 for some reason.
Author: Jani Monoses <jani@ubuntu.com>
Upstream-Bug: https://ghc.haskell.org/trac/ghc/ticket/5914

Index: b/m4/get_arm_isa.m4
===================================================================
--- a/m4/get_arm_isa.m4
+++ b/m4/get_arm_isa.m4
@@ -81,7 +81,7 @@ AC_DEFUN([GET_ARM_ISA],
                 fi],
                 [changequote(, )dnl
                  ARM_ISA=ARMv7
-                 ARM_ISA_EXT="[VFPv3,NEON]"
+                 ARM_ISA_EXT="[VFPv3D16,NEON]"
                  changequote([, ])dnl
                 ])
         ])
